package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-shopify-draggable",
-  "version": "0.1.0",
+  "version": "0.0.6",
```

Comment:
did you mean to downgrade the version # from 0.1.0 to 0.0.6?

Reply:
I did. Im trying a new way of releasing/publishing and accidentally had it go to 0.1.0